src/sqlancer/cnosdb/ast/CnosDBFunctionWithUnknownResult.java

```diff
@@ -0,0 +1,110 @@
+    RPAD3("rpad", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.INT, CnosDBDataType.STRING),
+    RPAD2("rpad", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.INT),
+    LTRIM("ltrim", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.STRING),
+    OCTET_LENGTH("octet_length", CnosDBDataType.INT, CnosDBDataType.STRING),
+    // REPEAT("repeat", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.INT),
+    REPLACE("replace", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.STRING),
+    REVERSE("reverse", CnosDBDataType.STRING, CnosDBDataType.STRING),
+    RIGHT("right", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.INT),
+    RTRIM("rtrim", CnosDBDataType.STRING, CnosDBDataType.STRING),
+    SPLIT_PART("split_part", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.INT),
+    STARTS_WITH("starts_with", CnosDBDataType.BOOLEAN, CnosDBDataType.STRING, CnosDBDataType.STRING),
+    STRPOS("strpos", CnosDBDataType.INT, CnosDBDataType.STRING, CnosDBDataType.STRING),
+    SUBSTR("substr", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.INT, CnosDBDataType.INT),
+    TRANSLATE("translate", CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.STRING, CnosDBDataType.STRING),
+
+    UPPER("upper", CnosDBDataType.STRING, CnosDBDataType.STRING),
+
+    MD5("md5", CnosDBDataType.STRING, CnosDBDataType.STRING),
+
+    // mathematical functions
+    ABS("abs", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    CEIL("ceil", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    EXP("exp", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE), LN("ln", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    LOG2("log2", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    LOG10("log10", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    POWER("power", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    ROUND("round", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    TRUNC("trunc", CnosDBDataType.DOUBLE, CnosDBDataType.INT),
+    FLOOR("floor", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    SIGNUM("signum", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    ACOS("acos", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    ASIN("asin", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    ATAN2("atan2", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    COS("cos", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE), SIN("sin", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    SQRT("sqrt", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    TAN("tan", CnosDBDataType.DOUBLE, CnosDBDataType.DOUBLE),
+    DATA_PART("date_part", CnosDBDataType.INT, CnosDBDataType.STRING, CnosDBDataType.TIMESTAMP);
+
+    // because bug of arrow-csv https://github.com/apache/arrow-rs/issues/3547
+    // NOW("now", CnosDBDataType.TIMESTAMP),
```

Comment:
As an optional suggestion, as in the other instance, we could apply the `CnosDBBugs` pattern here.

Reply:
Thanks, I remove those comment.